build-all-containers.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
```

Comment:
Same comment about '#!/bin/bash' as one below

Reply:
The other file was an unintentional change. I will leave this one as-is and revert the other.